Bootstraps/Symfony.php

```diff
@@ -133,11 +133,13 @@ public function postHandle($app)
         $container = $app->getContainer();
 
         if ($container->has('doctrine')) {
-            $em = $container->get("doctrine");
-            if (!$em->getManager()->isOpen()) {
-                $em->resetManager();
-            } else {
-                $em->getManager()->clear();
+            $doctrineRegistry = $container->get("doctrine");
+            foreach ($doctrineRegistry->getManagers() as $curManagerName => $curManager) {
+                if (!$curManager->isOpen()) {
+                    $doctrineRegistry->resetManager($curManagerName);
+                } else {
```

Comment:
Confused- can‘t you redet the manager directly and get rid of the name key?

Reply:
No, because its the registry's job to replace the manager if its no longer open (see https://github.com/doctrine/persistence/blob/master/lib/Doctrine/Common/Persistence/AbstractManagerRegistry.php#L217)